### ISSUE TITLE
Learn MVVM in Android.

### ISSUE THREAD

**Issue by YigitSekerci** [bounswe/bounswe2023group4#183]
### Problem Definition

MVVM is a widely used design pattern in android every team member should learn it to have well structured code. While learning MVVM every member should also learn its usage in android and how to implement it.

### Problem Context

* This issue is one of the divided issues for mobile research which are indicated in https://github.com/bounswe/bounswe2023group4/issues/176.

### Acceptance Criteria

* Learn what is MVVM
* Learn how to use MVVM in Android.

### Suggested Solutions

* https://www.youtube.com/@PhilippLackner This YouTube channel have many videos about mvvm and android team members can look it.

**Comment by YigitSekerci** [comment]
I have learned MVVM and MVI structures in android and learned their differences in implementation of android apps. I learned these topics from the YouTuber that I gave in suggested solution section.

**Comment by aemresafak** [comment]
Yeah, I am up to date with MVVM standards. I know what view is model is and view model is. I can apply the architecture.

**Comment by AliAlperenSonmez** [comment]
I have gained a solid understanding of the MVVM (Model-View-ViewModel) design pattern, recognizing its importance in achieving well-structured and maintainable Android code. I am proficient in applying the MVVM pattern within Android applications, allowing me to create organized and efficient software architectures that separate concerns effectively. I have leveraged valuable resources, such as the videos available on Philipp Lackner's YouTube channel, to learn and implement MVVM in Android, ensuring our team's codebase is designed with best practices in mind.
